docs/UserGuide.md

```diff
@@ -12,7 +12,8 @@ FriendDex is a **relationship management tool for CLI enthusiasts** looking to e
 
 ## Quick start
 
-1. Ensure you have Java `11` or above installed in your Computer.
+1. Ensure you have Java `11` or above installed in your Computer. Your Computer's operating system (OS) needs to be Windows, macOS or Linux. Another point to note is the OS version you are using should still be supported by the respective companies.
```

Comment:
Copy nit: "Your Computer's operating system (OS) needs to be Windows, macOS or Linux, and the OS version you are using should still be supported by the respective companies."